pkgs/development/libraries/libprom/default.nix

```
@@ -0,0 +1,39 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
+in
+stdenv.mkDerivation rec {
+  pname = "libprom";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "digitalocean";
+    repo = "prometheus-client-c";
+    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  doCheck = false;
+
+  patches = [
+    ./prometheus-client-c-0.1.1.patch
+  ];
+
+  preConfigure = ''
+    cd prom
+    export TEST=0
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/digitalocean/prometheus-client-c/";
+    description = "A Prometheus Client in C";
+    platforms = stdenv.lib.platforms.unix;
+    license = stdenv.lib.licenses.asl20;
+    maintainers = [ maintainers.cfsmp3 ];
```

You already use `stdenv.lib.*` almost everywhere in here, so might as well drop it from up here and use it actually-everywhere:

```suggestion
  meta = {
    homepage = "https://github.com/digitalocean/prometheus-client-c/";
    description = "A Prometheus Client in C";
    platforms = stdenv.lib.platforms.unix;
    license = stdenv.lib.licenses.asl20;
    maintainers = [ stdenv.lib.maintainers.cfsmp3 ];
```

---

Done.